src/assets/styles/damien-global.css

```diff
@@ -184,4 +184,7 @@ svg {
 }
 ul {
   list-style: none;
-}
\ No newline at end of file
+}
+#content a {
+  color: #337ab7;
+}
```

Comment:
@cesarvh The color applied to all `<a>` elements is configured in vuetify.ts - see `themes.light.anchor` and `themes.dark.anchor`. Currently it's set to a nice shade of orange - which probably won't look good on the yellow background of the service alert. 

Maybe make your CSS selector here more specific, so it only overrides the configured anchor color inside the service alert element.